admin/src/main/resources/io/buoyant/admin/js/src/delegator.js

```diff
@@ -67,6 +67,9 @@ define([
           obj.tree.transformation = obj.name;
           obj.tree.isTransformation = true;
           obj.bound.transformed = renderNode(obj.tree, primary);
+          if (ttype != success) {
+            obj.bound.addr.type = "neg";
+          }
```

Comment:
Good question.  A transformation object has 2 children: `obj.tree` which is the result of the transformation and `obj.bound` which is the bound address pre-transformation.  We show both so that we can see the effect of the transformation.  

In the case where the transformation results in a failure we want to color the whole path red, even if the pre-transformed value is valid.  That's what this does.  Without it, the pre-transformed value would be green while the rest of the path would be red.